src/main/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACL.java

```diff
@@ -283,7 +283,11 @@ private String getRepositoryName() {
         Describable scm = null;
         if (this.item instanceof WorkflowJob) {
             WorkflowJob project = (WorkflowJob) item;
-            scm = project.getProperty(BranchJobProperty.class).getBranch().getScm();
+            if (project.getProperty(BranchJobProperty.class) != null) {
```

Comment:
This is the wrong API. Do not depend on `workflow-multibranch`. `scm-api` should do what you need.

Reply:
The multibranch _folder_ (`MultiBranchProject`) containing the branch project (its `getParent()`) will be an `SCMSourceOwner`, if you are looking for an `SCMSource`. You do not need to depend on `branch-api` either.

If you are looking for an `SCM`, use `SCMTriggerItem.getSCMs`. This will work properly for both `AbstractProject` and `WorkflowJob`, without a `workflow-job` dep.

Reply:
Use `SCMTriggerItem.getSCMs` rather than an `instanceof` check, and beware those `get(0)` calls which suggest edge-case bugs.

`getSCMSources` returns a list of `SCMSource`s, not `SCM`s. Depends on what you are actually looking for and why.